CHANGES.md

```diff
@@ -1,10 +1,16 @@
 # Change Log
 
+# 0.6.0
```

Comment:
Looks like a tiny bugfix.
Let's please tag it as `0.5.3` patch version, per https://semver.org/

Reply:
Done. 0.5.3